scripts/ksubot.py

```diff
@@ -69,27 +69,25 @@ async def main():
     print("[+] Uploading to telegram")
     check_environ()
     print("[+] Files:", sys.argv[1:])
-    bot = telegram.Bot(BOT_TOKEN)
+    bot = telegram.Bot(BOT_TOKEN, base_url='http://127.0.0.1:8088/bot')
```

Comment:
check API_ID and fallback to api.telegram.org if this var is not set ?

Reply:
Added.